script/cli/build.rs

```diff
@@ -1,24 +1,46 @@
-use std::fs;
+use std::path::Path;
 use std::process::Command;
+use std::{env, fs};
 
 fn main() {
-    // Create assets directory
+    let clean = env::var("CLEAN").unwrap_or("false".to_string()) == "true";
```

Comment:
recommend using [structopt](https://docs.rs/structopt/latest/structopt/) to manage env variables / cli parameters